scripts/check-pristine-state.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const {
+    promisify
+} = require("util");
+const {
+    exec
+} = require('child_process');
+const execAsync = promisify(exec);
+
+const command = 'git status --porcelain | awk \'{print $2}\'';
+const [node, path, ...filesToIgnore] = process.argv;
+
+init();
```

Comment:
Matteo I know your a big fan of hoisting but `eslint-config-skyscanner` doesn't allow it. When we integrate it (soon) these will have to change anyways.

Reply:
😢